tests/cypress/e2e/hedy_page/invalid_program.cy.js

```diff
@@ -0,0 +1,58 @@
+import {goToHedyLevel5Page, goToHedyPage} from "../tools/navigation/nav";
+
+describe('Error code gives correct error', () => {
+    it('Misspelled Keyword', () => {
+        const error_code = "prnt Hello world!"
+        const error_code2 = "print Hello world!\nak Hello world?"
```

Comment:
I think perhaps these could be 2 different tests? You can change the `it` for a describe and then have two it within that one. We do something like that in the lezer tests.